components/SideNavigation.vue

```diff
@@ -3,25 +3,25 @@
     <div class="SideNavigation-HeadingContainer sp-flex">
       <v-icon
         class="SideNavigation-HeadingIcon pc-none"
-        :aria-label="$t('Navi Open')"
+        :aria-label="$t('サイドメニュー項目を開く')"
         @click="openNavi"
       >
         mdi-menu
       </v-icon>
       <nuxt-link to="/" class="SideNavigation-HeadingLink">
```

Comment:
ここ（「東京都」のロゴをクリックしたときのリンク先）は，日本語トップページより各言語のトップページの方が相応しく，
```suggestion
      <nuxt-link :to="localePath('/')" class="SideNavigation-HeadingLink">
```
とした方がよいと思うのですが，いかがでしょう．